core/modules/wiki.js

```diff
@@ -190,15 +190,15 @@ exports.getChangeCount = function(title) {
 
 /*
 Generate an unused title from the specified base
+options.prefix must be a string
 */
 exports.generateNewTitle = function(baseTitle,options) {
 	options = options || {};
+	options.prefix = (typeof(options.prefix) === "string") ? options.prefix : " ";
```

Comment:
Rather than assigning back to the callers object, we would usually code this as:

```
var prefix = (typeof(options.prefix) === "string") ? options.prefix : " ";
```